.gitattributes

```diff
@@ -1 +1 @@
-*.xz filter=lfs diff=lfs merge=lfs -text
+data/library_coverage_xml_and_fulltext_indicators.db.xz filter=lfs diff=lfs merge=lfs -text
```

Comment:
Why did you change this to not track `data/library_coverage_xml_and_fulltext_indicators.tsv.xz` using Git LFS? I think tracking all `.xz` files with LFS makes the most sense?

Reply:
A combination of oversight and confusion about what we agreed to track with `lfs` : P Reverted in 44aba2d.